puppet/modules/profiles/manifests/web.pp

```diff
@@ -62,7 +62,6 @@
   contain web::vhost::yum
 
   class { 'web::vhost::stagingyum':
```

Comment:
If you don't set parameters, the `contain` below is sufficient because it implies including the class. Note we do the same with `web::vhost::stagingdeb`.